### PR TITLE
GN-3766 - The overview page of frontend-rb should be sorted by default

### PR DESCRIPTION
**Pull request by usrtim**
I added to the list page of regulatory statements, following functionalities:
    - sort on modified date by default
    - include the modified and published dates
    - allow filtering on title
    - support pagination

Please let me know If I'm missing something? Thanks

### CODE REVIEW

**Comment by usrtim** [comment]
added functionality also for code list page. 

TODO: on code list I can't get **createdOn** date. Can we do something to fill those data ?

**Comment by usrtim** [comment]
> added functionality also for code list page.
> 
> TODO: on code list I can't get **createdOn** date. Can we do something to fill those data ? After that I would like to allow users to filter based on createOn date

UPDATE: this is done.

**Comment by elpoelma** [comment]
I updated the translations a bit. Sorting on title, created-on and updated-on works well. The default sort also seems to work as expected. Codelist sort with the new created-on property works as expected too. The publish-date does not work yet, but I suggest we wait for the PRs of @lagartoverde until merging this so we can update this PR to the new model.

File: app/templates/list.hbs
Comment (elpoelma) [inline app/templates/list.hbs:45]: The field property should probably be different here, but this too will change with the new model soon.

**Review by elpoelma — CHANGES_REQUESTED** [review]
While pagination seems to work, the pagination indicator at the bottom (showing the page number) does not seem to function correctly.

**Comment by elpoelma** [comment]
Additionally, did you look into adding a search field in order to filter the regulatory statements on title?

**Comment by usrtim** [comment]
> 

@elpoelma thanks for the comment. Yes I think we need to wait for the PR of @lagartoverde to be merged.

**Comment by usrtim** [comment]
> 

@elpoelma no, I didn't add search field to filter the regulatory statements on title. I didn't saw that on task description and for that reason I skipped that. I will add it.

File: app/templates/list.hbs
Comment (usrtim) [reply app/templates/list.hbs:45]: @elpoelma ok, will wait till then. Please let me know what should be the field property after we have new model

**Comment by usrtim** [comment]
> While pagination seems to work, the pagination indicator at the bottom (showing the page number) does not seem to function correctly.

We fixed that, seemed to be a backend issue. @elpoelma thanks .

**Pull request merged**
